scripts/examples/typescript/tests/getXapiVersion.ts

```diff
@@ -0,0 +1,28 @@
+import { xapi_client } from "../src/XenAPI"
+
+// Ref https://xapi-project.github.io/xen-api/
+async function get_api_version(session: any) {
+    const pool = await session.xenapi.pool.get_all()
+    const host = await session.xenapi.pool.get_master(pool[0])
+    const major = await session.xenapi.host.get_API_version_major(host)
+    const minor = await session.xenapi.host.get_API_version_minor(host)
+    return `${major}.${minor}`
+}
+
+async function main() {
```

Comment:
This example doesn't have any error handling. Another idea could be to move the body of the function into a try-finally where the finally block would contain the session logout - this would be educational as it would emphasize that we should always log out our sessions before exiting the program.